Plugins/src/main/java/tw/waterball/judgegirl/plugins/api/codeinspection/CodeQualityInspectionReport.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 Johnny850807 (Waterball) 潘冠辰
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package tw.waterball.judgegirl.plugins.api.codeinspection;
+
+import lombok.Value;
+
+/**
+ * @author - johnny850807@gmail.com (Waterball)
+ */
+@Value
+public class CodeQualityInspectionReport {
+    CyclomaticComplexityReport CCReport;
```

Comment:
Still should be camelCase even it's an acronym. Name it `ccReport`.

And this should be a private attribute.